lua/gatekeeper.lua

```diff
@@ -155,6 +155,20 @@ struct gt_config {
 	/* This struct has hidden fields. */
 };
 
+typedef long time_t;
```

Comment:
I think `time_t` can change depending on where/how the code is compiled. It'd be safer to have a C function that receives a pointer `struct timeval *` and the parameter `sec` and `usec`, and sets the struct properly. This way, all that LuaJIT will need to know is "struct timeval;".